BUILDING.md

```diff
@@ -131,3 +131,34 @@ When working with `ctr`, the containerd CLI we just built, don't forget to start
 ```sh
 containerd --config config.toml
 ```
+
+# Testing containerd
+
+During the automated CI the unit tests and integration tests are run as part of the PR validation. As a developer you can run these tests locally by using any of the following `Makefile` targets:
+ - `make test`: run all non-integration tests that do not require `root` privileges
+ - `make root-test`: run all non-integration tests which require `root`
+ - `make integration`: run all tests, including integration tests and those which require `root`
+ - `make integration-parallel`: run all tests (integration and root-required included) in parallel mode
+
```

Comment:
Perhaps add how to run an individual test as root? 
`go test -v -run "<TEST_NAME>" . -test.root`

or simply that `-test.root` flag is an option

or where to find these flags.

Reply:
Added a few sentences and examples for running `go test` directly. PTAL